outputs.tf

```diff
@@ -1,4 +1,25 @@
-output "ssm_document_name" {
-  description = "Name of the AWS SSM Document that setups the Lacework agent"
-  value       = aws_ssm_document.setup_lacework_agent.name
+output "external_id" {
+  value       = local.iam_role_external_id
+  description = "The External ID configured into the IAM role"
+}
+
+output "iam_role_name" {
+  value       = var.iam_role_name
```

Comment:
@afiune does this need to be `var.iam_role_name` or `local.iam_role_name`?

Reply:
YES YES AND YES!! Thank you, great catch!